test/TestApps/PushNotificationsTestApp/main.cpp

```diff
@@ -280,13 +286,14 @@ bool RegisterActivator()
     return true;
 }
 
-bool UnregisterActivator()
+bool UnregisterActivator(winrt::Microsoft::Windows::PushNotifications::PushNotificationRegistrationToken token)
 {
+    PushNotificationManager::UnregisterActivator(token, PushNotificationRegistrationKind::PushTrigger | PushNotificationRegistrationKind::ComActivator);
     try
     {
         PushNotificationActivationInfo info(
             PushNotificationRegistrationKind::PushTrigger | PushNotificationRegistrationKind::ComActivator,
-            winrt::guid("00000000-0000-0000-0000-000000000001"));
+            c_fakeComServerId);
 
         auto token = PushNotificationManager::RegisterActivator(info);
```

Comment:
Bit confused as to why we need a RegisterActivator in an UnREgisterActivator test